src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -328,11 +328,11 @@
   <area alias="media">
     <key alias="clickToUpload">Klik for at uploade</key>
     <key alias="orClickHereToUpload">eller klik her for at vælge filer</key>
-    <key alias="disallowedFileType">Kan ikke uploade denne fil, den har ikke en godkendt filtype</key>
-    <key alias="maxFileSize">Maks filstørrelse er</key>
+    <key alias="disallowedFileType">Kan ikke uploade denne fil; den har ikke en godkendt filtype</key>
+    <key alias="maxFileSize">Maksimum filstørrelse er</key>
```

Comment:
Maybe use `Maksimal` instead of `Maksimum`. Think that is a bit more correct.

Reply:
Agreed 👍

Reply:
Although, we're using "Maksimum" in quite a lot of other places... 🤔